integrations/shopify/src/actions/get-customerOrders.ts

```diff
@@ -0,0 +1,33 @@
+import { SHOPIFY_API_VERSION } from '../const'
+import { prepareParams } from '../utils'
+import axios from 'axios'
+import * as botpress from '.botpress'
+
+type GetCustomerOrders = botpress.IntegrationProps['actions']['getCustomerOrders']
+
+export const getCustomerOrders: GetCustomerOrders = async ({ ctx, input, logger }) => {
+  const { customer_id, status } = input
+  const filters = await prepareParams({ customer_id, status })
+
+  axios.defaults.baseURL = `https://${ctx.configuration.shopName}.myshopify.com`
+  axios.defaults.headers['X-Shopify-Access-Token'] = ctx.configuration.access_token
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏ Do not set any global axios config since integrations are multi tenant